packages/strapi-plugin-i18n/admin/src/hooks/useDeleteLocale/reducer.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable consistent-return */
+import produce from 'immer';
+
+export const initialState = {
+  localeToDelete: null,
+  isDeleteModalOpen: false,
+  isDeleting: false,
+};
+
+export const Actions = {
```

Comment:
Can you put this in a file called `constants.js` so it stays consistent with the current file architecture?

```js
export const SHOW_MODAL = 'i18n.SHOW_MODAL;
```